src/main/java/no/unit/nva/doi/transformer/language/LanguageMapper.java

```diff
@@ -33,26 +32,12 @@ private LanguageMapper() {
     /**
      * Map an ISO639-3 language identifier to a Language URI.
      *
-     * @param iso6393 An ISO639-3 identifier.
+     * @param iso An ISO639-3 identifier.
      * @return a language URI if this mapping is available or an empty {@link Optional} if there is no such mapping.
      */
-    public static Optional<URI> getUriFromIso639AsOptional(String iso6393) {
```

Comment:
the reverse mapping does not work right now, but we are not using it at the moment